src/App.js

```diff
@@ -0,0 +1,22 @@
+import { useDispatch, useSelector } from "react-redux";
+import {
+  addToCart,
+  removeFromCart,
+  setTotalAmount,
+  setCategoryFilter,
+  setPriceFilter,
+  setSortedFilter,
+  setTheme,
+} from "./redux/actions";
+import { Flex, Select, Text, Button, toast } from "@sparrowengg/twigs-react";
+
+function App() {
+  const dispatch = useDispatch();
+  return (
+    <>
+      <Text css={{ backgroundColor: "$primary" }}>this is a sample text</Text>
```

Comment:
@arun-121 is this file needed???

Reply:
@arun-121 why do we have only one component here

Reply:
removed the same